lib/auth0_ex/parser.ex

```diff
@@ -38,9 +38,19 @@ defmodule Auth0Ex.Parser do
     end
   end
 
-  defp is_json_content_type?(headers),
-    do:
-      headers
-      |> Map.new()
-      |> Map.get("Content-Type") == "application/json"
+  defp is_json_content_type?(headers) do
```

Comment:
Usually with predicate methods you omit `is_foo?` and just make it `foo?`.

Reply:
definitely agreed.